soc/hps_soc.py

```diff
@@ -221,11 +221,14 @@ def main():
                         help="Use slimmer VexRiscv (required for mnv2_first)")
     parser.add_argument("--build", action="store_true",
                         help="Whether to do a full build, including the bitstream")
-    parser.add_argument("--toolchain", default="radiant",
-                        help="Which toolchain to use, radiant (default) or oxide")
+    parser.add_argument("--toolchain", default="oxide",
+                        help="Which toolchain to use: oxide (default) or radiant")
     parser.add_argument("--synth_mode", default="radiant",
-                        help="Which synthesis to use, radiant/synplify (default), lse, or yosys")
-    parser.add_argument("--litespi-flash", action="store_true", help="Use litespi flash")
+                        help="Which synthesis mode to use with Radiant toolchain: "
+                        "radiant/synplify (default), lse, or yosys")
+    parser.add_argument("--no-litespi-flash", dest="litespi_flash",
```

Comment:
Ok, I see there is precedent in LiteX for `--no-X` options.

Reply:
Yes the `--no-x` convention is pretty common. I think having `--litespi-flash=False` would be a bit weird. I would be happy to change it to `--old-flash`, that's probably a good name because it looks like Florent is intending to fully deprecate that code in favour of Litespi.

But if you're okay with it, I'll err on the side of laziness :-) and leave this as `--no-litespi-flash` because we already have that option in hps-firmware.git currently and I would like to stay consistent.